.github/scripts/validate-model-manifest.py

```
@@ -0,0 +1,111 @@
+)
+
+REQUIRED_MODEL_FIELDS = {"id", "label", "vertexId", "provider", "available"}
+
+
+def fail(msg: str) -> None:
+    print(f"FAIL: {msg}", file=sys.stderr)
+    sys.exit(1)
+
+
+def load_committed_manifest() -> dict:
+    result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)